### DISCUSSION TITLE
v5.1

### DISCUSSION THREAD

**Discussion by mstv** [gitextensions/gitextensions#12015] [General]
Is there anything else to be added to v5.1 (except the three ready bugfixes in https://github.com/gitextensions/gitextensions/milestone/73)?

**Comment by RussKie** [comment]
None from my POV. I'm happy to cut a release when you tell me.

**Reply by gerhardol** [reply]
Go with a release from master, not just 5.1 MS
All PRs merged. 
Doc to be updated as usual.

**Reply by mstv** [reply]
> I'm happy to cut a release when you tell me.
> All PRs merged.

Thank you very much!

**Reply by RussKie** [reply]
@gitextensions/git-extensions-team may I get you to come up with release notes (highlights) and add those to the release draft?

**Reply by mstv** [reply]
I have added a proposal to the v5.1 draft at https://github.com/gitextensions/gitextensions/releases.

**Reply by gerhardol** [reply]
> I have added a proposal to the v5.1 draft at https://github.com/gitextensions/gitextensions/releases.

looks fine

**Reply by RussKie** [reply]
Thanks you, folks.

**Reply by RussKie** [reply]
Done. 
If there're no catastrophic failures reported in the next few days, I'll update the `configdata` branch.
